extensions/amp-next-page/1.0/test/validator-amp-next-page-remote.out

```diff
@@ -0,0 +1,60 @@
+FAIL
+|  <!doctype html>
+|  <html ⚡>
+|  <head>
+|    <meta charset="utf-8">
+|    <title>AMP next page examples</title>
+|    <link rel="canonical" href="amps.html">
+|    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+|    <link href="https://fonts.googleapis.com/css?family=Questrial" rel="stylesheet" type="text/css">
+|    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+|    <script async src="https://cdn.ampproject.org/v0.js"></script>
+|    <script async custom-element="amp-next-page" src="https://cdn.ampproject.org/v0/amp-next-page-1.0.js"></script>
+|    <style amp-custom>
+|      .amp-next-page-text {
+|        font-family: serif;
+|      }
+|    </style>
+|  </head>
+|  <body>
+|    <h2>Content discovery</h2>
+|      <!-- Valid: Correct use of amp-next-page with an src and xssi-prefix -->
+|      <amp-next-page src="https://foo.com/data.json" xssi-prefix=")]}" deep-parsing="false" max-pages="5">
+|      <script type="application/json">
+|        [
+|          {
+|            "image": "/examples/img/hero@1x.jpg",
+|            "title": "Page 1",
+|            "url": "./articles/article-short.html"
+|          },
+|          {
+|            "image": "/examples/img/hero@1x.jpg",
+|            "title": "Page 2",
+|            "url": "./articles/article-with-fixed.html"
+|          },
+|          {
+|            "image": "/examples/img/hero@1x.jpg",
+|            "title": "Page 3",
+|            "url": "./articles/article-long.html"
+|          },
+|          {
+|            "image": "/examples/img/hero@1x.jpg",
+|            "title": "Page 4",
+|            "url": "./amp-next-page.amp.html"
+|          }
+|        ]
+|      </script>
+|      <div separator class="my-custom-separator">
+|        Separator
+|      </div>
+|      <div footer>
+|        Next page recommendation box
+|      </div>
+|      <footer>
+>>     ^~~~~~~~~
+amp-next-page/1.0/test/validator-amp-next-page-remote.html:52:4 The tag 'footer', a child tag of 'amp-next-page with src attribute', does not satisfy one of the acceptable reference points: AMP-NEXT-PAGE > [separator], AMP-NEXT-PAGE > [footer], AMP-NEXT-PAGE > SCRIPT[type=application/json]. (see https://amp.dev/documentation/components/amp-next-page)
```

Comment:
@honeybadgerdontcare  or @Gregable , what's a good way to do this: Allow arbitrary content inside `<amp-next-page>` while also validating reference points that are either required or unique?